pkg/endpointmanager/manager.go

```diff
@@ -20,6 +20,7 @@ import (
 	"sync"
 
 	"github.com/cilium/cilium/pkg/endpoint"
+	endpointid "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the id package commit in order not to break git bisecting as the previous commit alone will not compile.